modules/api/lambda/package.json

```diff
@@ -34,18 +34,19 @@
     "uuid": "^8.3.2"
   },
   "devDependencies": {
-    "@types/cookie": "^0.4.0",
-    "@types/express": "^4.17.11",
-    "@types/glob": "^7.1.3",
-    "@types/morgan": "^1.9.2",
+    "@types/cookie": "^0.4.1",
+    "@types/express": "^4.17.13",
+    "@types/glob": "^7.1.4",
+    "@types/morgan": "^1.9.3",
+    "@types/node": "^16.3.3",
```

Comment:
`@types/node` should be updated here:

https://github.com/squidfunk/terraform-aws-cognito-auth/blob/a3298290cf0f430ea88184743521f50794d7a5c1/package.json#L27

Reply:
Done.